CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Remove diagnostics module. ([PR #60](https://github.com/ClimateImpactLab/dscim/pull/60, [@JMGilbert](https://github.com/JMGilbert))
 - Remove old/unnecessary files. ([PR #57](https://github.com/ClimateImpactLab/dscim/pull/57), [@JMGilbert](https://github.com/JMGilbert))
 
```

Comment:
does it make sense to add mention of pinning the two packages due to the float issue?

Reply:
I'm working to unpin the packages by messing with the code, so I'm hoping that won't be necessary. But I will definitely mention that if I can't figure this out.